components/core/buttons/TextButton.vue

```diff
@@ -1,9 +1,14 @@
 <template>
-    <button :class="coreButtonClasses">
-        <ext-link v-if="href" :href="href">
+    <button>
+        <ext-link v-if="href" :href="href" :class="coreButtonClasses">
             <slot></slot>
         </ext-link>
-        <locale-link v-else-if="to" :to="to" customized>
+        <locale-link
+            v-else-if="to"
+            :to="to"
+            customized
+            :class="coreButtonClasses"
+        >
             <slot></slot>
         </locale-link>
         <slot v-else></slot>
```

Comment:
Thanks for submitting the PR. Here's my feedback:
1. Look at the `v-if`, `v-else-if`, and `v-else` - the last case wasn't covered by the changes.
2. The changes made them behave ideally in the cases of `<ext-link>` and `<locale-link>`. However, adding `coreButtonClasses` onto them isn't a sensible way to do it. Why? Because essentially, in the scope of a component, the links shouldn't represent the class name `core-button` while having the existing parent element `<button>`. In the meantime, they indeed should be applied with the `.core-button` CSS rules.

A more elegant way would be, in my opinion:
1. Use javascript to determine if it's one of the link types (`ext-link` or `locale-link`)
2. Add a `--is-link` class in the `coreButtonClasses` according to 1.
3. Replace the original CSS selector `.core-button {...} ` with `.core-button:not(.--is-link), .corebutton.--is-link > a { ... }` to adhere to the existing CSS styling.

This way, three cases of the `core-button` variants are covered.

Please DM me if the above description didn't explain well. We could go through this in a short talk.